androidApp/src/androidMain/kotlin/com/macaosoftware/sdui/app/MainActivity.kt

```diff
@@ -23,6 +25,7 @@ class MainActivity : ComponentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        Firebase.initialize(this)
```

Comment:
I think is better not to call this directly here but doing it through the plugin interface. Also I wonder if there is a way to initialize specific Firebase features, like Firebase authorization only. We don't want to use all the features Firebase have.